# Translations template for vigilo-connector.
# Copyright (C) 2011 ORGANIZATION
# This file is distributed under the same license as the vigilo-connector
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-connector 2.0.1\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2011-06-09 15:52+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/connector/client.py:54
msgid ""
"Use 'require_tls'. 'require_compression' option is ignored when both "
"'require_*' options are True."
msgstr ""

#: src/vigilo/connector/client.py:80
#, python-format
msgid "Authentication failure: %s"
msgstr ""

#: src/vigilo/connector/client.py:85
msgid "Server does not support TLS encryption."
msgstr ""

#: src/vigilo/connector/client.py:156
msgid ""
"The \"watched_topics\" option has now been renamed into \"subscriptions\". "
"Please update your configuration file."
msgstr ""

#: src/vigilo/connector/converttoxml.py:57 src/vigilo/connector/converttoxml.py:64
#: src/vigilo/connector/converttoxml.py:70
#, python-format
msgid "Unknown/malformed message type: '%s'"
msgstr ""

#: src/vigilo/connector/converttoxml.py:74
#, python-format
msgid "Got malformed message: %s"
msgstr ""

#: src/vigilo/connector/forwarder.py:35
msgid "no connection"
msgstr ""

#: src/vigilo/connector/forwarder.py:39
msgid "no connection to the XMPP server"
msgstr ""

#: src/vigilo/connector/forwarder.py:106
#, python-format
msgid ""
"Can't understand the max_queue_size option, it should be an integer (or 0 for"
" no limit). Current value: %s"
msgstr ""

#: src/vigilo/connector/forwarder.py:114
msgid "The max_queue_size option is only available on Python >= 2.6. Ignoring."
msgstr ""

#: src/vigilo/connector/forwarder.py:126
msgid "Connected to the XMPP bus"
msgstr ""

#: src/vigilo/connector/forwarder.py:145
#, python-format
msgid "Lost connection to the XMPP bus (reason: %s)"
msgstr ""

#: src/vigilo/connector/forwarder.py:177
#, python-format
msgid "Unable to forward the message (%(reason)s)"
msgstr ""

#: src/vigilo/connector/forwarder.py:183
msgid ". it has been stored for later retransmission"
msgstr ""

#: src/vigilo/connector/forwarder.py:256
#, python-format
msgid "Batch sent, waiting for %d replies from the bus"
msgstr ""

#: src/vigilo/connector/forwarder.py:273
#, python-format
msgid "Error trying to save a message to the backup database: %s"
msgstr ""

#: src/vigilo/connector/forwarder.py:452
#, python-format
msgid "No destination node configured for messages of type '%s'. Skipping."
msgstr ""

#: src/vigilo/connector/main.py:42
#, python-format
msgid "Directory not found: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/main.py:47
#, python-format
msgid "Directory not readable: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/main.py:52
#, python-format
msgid "Directory not writable: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/main.py:57
#, python-format
msgid "Directory not executable: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/nodetoqueuefw.py:62
msgid "Incoming message is too big to be stored in the queue, dropping it."
msgstr ""

#: src/vigilo/connector/nodetoqueuefw.py:67
#, python-format
msgid "The queue is full, dropping incoming XML message! (%s)"
msgstr ""

#: src/vigilo/connector/nodetosocketfw.py:26
msgid "socket not connected"
msgstr ""

#: src/vigilo/connector/options.py:13
msgid "Choose the service name"
msgstr ""

#: src/vigilo/connector/options.py:14
msgid "Load this settings.ini file"
msgstr ""

#: src/vigilo/connector/options.py:25
msgid "The configuration file does not exist"
msgstr ""

#: src/vigilo/connector/presence.py:98
#, python-format
msgid "Queue size too high (%s) ! Switching presence to unavailable."
msgstr ""

#: src/vigilo/connector/presence.py:139
#, python-format
msgid ""
"Another instance of %(user)s@%(host)s has priority %(priority)d ! (me: "
"%(me)s, other: %(other)s)"
msgstr ""

#: src/vigilo/connector/queuetonodefw.py:86
msgid "Received request to shutdown"
msgstr ""

#: src/vigilo/connector/queuetonodefw.py:91
msgid "Item is None in consumeQueue, this should never happen!"
msgstr ""

#: src/vigilo/connector/queuetonodefw.py:96
msgid "Stopping QueueToNodeForwarder."
msgstr ""

#: src/vigilo/connector/status.py:126
#, python-format
msgid "Stats for %(service)s: %(name)s = %(value)s"
msgstr ""

#: src/vigilo/connector/store.py:169 src/vigilo/connector/store.py:264
#, python-format
msgid "Could not fill the output buffer: %s"
msgstr ""

#: src/vigilo/connector/store.py:209
#, python-format
msgid "Could not VACUUM the database: %s"
msgstr ""

